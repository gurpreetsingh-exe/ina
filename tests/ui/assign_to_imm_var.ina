// COMPILE_FAIL

fn main() {
    let a = 20;
    a = 50;
}
