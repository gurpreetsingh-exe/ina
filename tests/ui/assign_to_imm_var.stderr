error: assignment to immutable variable
   ╭─[tests/ui/assign_to_imm_var.ina:5:5]
   │
 4 │     let a = 20;
   │     ─────┬─────
   │          ╰────── `a` defined here
   ∙
 5 │     a = 50;
   │     ┬
   │     ╰─ cannot assign to immutable variable `a`
   ∙
───╯
